modules/apps/site-initializer/site-initializer-liferay-marketplace/site-initializer-liferay-marketplace-test/src/testFunctional/paths/MP2Navigation.path

```diff
@@ -0,0 +1,22 @@
+<html>
+<head>
+<title>MP2Navigation</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">MP2Navigation</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>NAV_ITEM</td>
+	<td>//*[contains(@class, 'dashboard-navigation-body-list')]/following-sibling::div//*[contains(.,'${key_navItem}')]</td>
```

Comment:
Hey @LuanaKSilva, this path `NAV_ITEM` can be simpler, for example: `//*[contains(@class, 'dashboard-navigation-body-list') and contains(text(),'${key_navItem}')]` . It is up to you to change or not.

Reply:
This is a good way, but the path is not so solid. Thanks for the suggestion.